en/computer_vision/path_planning_interface.md

```diff
@@ -107,6 +108,17 @@ PX4 safely handles the case where messages are not received from the offboard sy
     - A planner will mirror back setpoints it receives when the vehicle is in a mode/state for which it doesn't provide path planning. (i.e. the vehicle will follow its desired path, delayed by a very small amount). 
 
 
+## Bezier Trajectory Interface {#bezier_interface)
+
+If the obstacle avoidance interface is enabled, it can also receive mavlink messages in the [TRAJECTORY_REPRESENTATION_BEZIER](https://mavlink.io/en/messages/common.html#TRAJECTORY_REPRESENTATION_BEZIER) format. This is parsed as follows:
+
+- The number of Bezier control points determines the degree of the Bezier curve. For example, 3 points makes a quadratic bezier curve with constant acceleration.
+- The Bezier curve must be the same degree in x,y,z and yaw, with all Bezier control points finite
+- The `delta` array should have the value corresponding with the last Bezier control point indicate the duration that the waypoint takes to execute the curve to that point, from beginning to end. Other values in the `delta` array are ignored.
+- The timestamp of the mavlink message should be the time that the curve starts, and communication delay and clock mismatch will be compensated for on the flight controller via the timesync mechanism.
+- The control points should all be specified in local coordinates.
```

Comment:
This means FRD relative to vehicle center-of-gravity and as shown here right?: https://docs.px4.io/master/en/getting_started/px4_basic_concepts.html#heading-and-directions ?

Reply:
No, it is NED. This is the same frame as the waypoint trajectory message, BTW.